lib/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { NextAction } from '@prisma/client'
 import { Decimal } from '@prisma/client/runtime/binary'
+import _ from 'lodash'
```

Comment:
```suggestion
import capitalize from 'lodash'
```

Reply:
Re-imported in 0ab3615d8bbd217d7b6e8a424da4612ad9896811